engine/src/main/java/org/destinationsol/game/SolCam.java

```diff
@@ -74,8 +76,13 @@ public void update(SolGame game, float timeStep) {
             return;
         }
 
+
         Hero hero = game.getHero();
         float life = hero.getLife();
+        Hull hull = null;
+        if(hero.isNonTranscendent())
```

Comment:
```suggestion
        if (hero.isNonTranscendent())
```
Also please use curly braces even if the block is a single line